de-lib/src/main/java/org/iplantc/de/client/services/impl/AppUserServiceFacadeImpl.java

```diff
@@ -175,10 +175,7 @@ public void getAppDetails(final App app, DECallback<App> callback) {
         deServiceFacade.getServiceData(wrapper, new DECallbackConverter<String, App>(callback) {
             @Override
             protected App convertFrom(String object) {
-                Splittable split = StringQuoter.split(object);
-                AutoBean<App> appAutoBean = AutoBeanUtils.getAutoBean(app);
-                AutoBeanCodex.decodeInto(split, appAutoBean);
-                return appAutoBean.as();
+                return AutoBeanCodex.decode(factory, App.class, object).as();
```

Comment:
Side note, I remembered the date parsing error.  It's that silly `NumberFormatException` that always comes up if you have dev tools open with `Pause on Caught Exceptions` enabled.  It doesn't show up in the console or the logs, but while I was trying to trace down all the various bugs, this was getting in my way enough that I thought I'd try to fix it.  So, it's really just an annoyance.